src/autoreload.js

```diff
@@ -0,0 +1,9 @@
+(function () {
+    var ws = new WebSocket('ws://' + window.location.host);
```

Comment:
We should make this connect to the dev server at a location which is not likely to conflict with users' proxy definitions. Something like `'ws://' + window.location.host + '/_trunk/ws'`.

When configured to inject the autoreloader, we can listen for WS connections at that location. Thoughts?